src/Pages/Goals.cshtml

```diff
@@ -41,7 +41,7 @@ else
                 <h3 class="text-light problematic-title" title="Title">@item.Title </h3>
                 <div class="progress mt-4">
                     @{
-                        var percent = decimal.Round(Model.Savings / item.Amount * 100);
+                        var percent = item.Amount == 0 ? 0 : decimal.Round(Model.Savings / item.Amount * 100);
                         percent = percent > 100 ? 100 : percent;
                         var color = percent < 100 ? (percent < 30? "danger" : (percent < 85 ? "warning" : "info" )) : "success";
```

Comment:
Space missing after '30' and before '?'.
Also I was thinking, should we declare how many of these ?: statements should we stack in one line? Should two be the limit? Because after we use them too much they become less clear and an switch statement would be clearer

Reply:
Yes, this looked like a good idea, because I wanted to add 3rd color, then when I added 4th it started to get messy, I think I'll use switch here.